packages/mui-joy/src/Avatar/avatarClasses.ts

```diff
@@ -3,6 +3,8 @@ import { generateUtilityClass, generateUtilityClasses } from '../className';
 export interface AvatarClasses {
   /** Class name applied to the root element. */
   root: string;
+  /** Styles applied to the root element if `color="default"`. */
+  colorDefault: string;
```

Comment:
why did you add this class?

Reply:
@hbjORbj 
colorDefault is used in a test case, please check below image

![image](https://user-images.githubusercontent.com/11643426/235900198-1ff6840a-896b-4e7b-b982-f4e00079f849.png)

Reply:
We can remove this line in the test case. We shouldn't add `colorDefault` to class names of Joy Avatar and document it in API. I will commit the change.

Reply:
Removed